packages/cms/src/components/Profile.jsx

```diff
@@ -27,10 +27,20 @@ class Profile extends Component {
       profile: props.profile,
       selectors: {},
       loading: false,
+      isIE: false,
       setVarsLoading: false
     };
   }
 
+  // IE check needed for position: sticky fallback
+  componentDidMount() {
+    if (typeof window !== "undefined") {
+      if (/*@cc_on!@*/false || !!document.documentMode) { // eslint-disable-line spaced-comment
+        this.setState({isIE: true});
+      }
+    }
+  }
+
```

Comment:
Having this in the `componentDidMount` means that IE will render once without this, and then after the component mounts it will trigger this and re-render, causing an additional render cycle in IE. The React lifecycle is:
* `constructor`
* `render`
* `componentDidMount`

Moving this to the `constructor` would solve this (no need to initialize it as `false`), but I think a better solution would be to do this detection completely outside of the component, way up at the top of the file with the imports (ie. `const isIE` instead of using state)